source/ui/CLS/CLSBeamlineStatusView.h

```diff
@@ -31,6 +31,8 @@ class CLSBeamlineStatusView : public QWidget
 	void beamStatusChanged(CLSBeamlineStatus *newStatus);
 	/// Notifier that the selected component has changed.
 	void selectedComponentChanged(AMControl *newControl);
```

Comment:
I think this signal should the the same as the new added signal `controlClicked()`. I think we need only one.